content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Notation v1.0.0-RC.7 is available!
+author:  "Notation Release Team"
+date: 2023-05-28
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.7, including [Notation CLI](https://github.com/notaryproject/notation) v1.0.0-rc.7, [notation-go library](https://github.com/notaryproject/notation-go) v1.0.0-rc.6, and [notation-core-go library](https://github.com/notaryproject/notation-go) v1.0.0-rc.4. This blog walks you through the updates in this release.
+
+> NOTE: Both Notation CLI v1.0.0-rc.7 and v1.0.0-rc.6 have the same functionalities. However, v1.0.0-rc.7 included an additional fix for an E2E test case.
+
+## What's new
+
+This release adds the following major changes:
+
+- Fix security advisories
```

Comment:
```suggestion
- Security advisory fixes
```

Also should we call out the list of advisisories?

Reply:
@yizha1 Let's add advisories in a follow-up PR.